src/Mod/PartDesign/Gui/TaskPadParameters.cpp

```diff
@@ -476,6 +621,15 @@ bool   TaskPadParameters::getCustom(void) const
     return ui->groupBoxDirection->isChecked();
 }
 
+std::string TaskPadParameters::getReferenceAxis(void) const
+{
+    std::vector<std::string> sub;
+    App::DocumentObject* obj;
+    getReferenceAxis(obj, sub);
+    std::string axis = buildLinkSingleSubPythonStr(obj, sub);
+    return axis;

```

Comment:
There's not really any reason to create the temporary here. I'd guess most compilers will figure that out, but it wouldn't hurt to combine these lines to make sure you get RVO.

Reply:
thanks, fixed.